statusHandler/termuic/termuiRenders/widgetsRender.go

```diff
@@ -184,14 +191,47 @@ func (wr *WidgetsRender) prepareChainInfo() {
 	wr.chainInfo.Rows = rows
 }
 
+func (wr *WidgetsRender) prepareBlockInfo() {
+	//7 rows and one column
+	numRows := 7
+	rows := make([][]string, numRows)
+
+	currentBlockHeight := wr.getFromCacheAsUint64(core.MetricSynchronizedRound)
+	//currentBlockHeight is equals with MetricSynchronizedRound + 1
+	currentBlockHeight++
+	rows[0] = []string{fmt.Sprintf("Current block height: %v", currentBlockHeight)}
+
+	numTransactionInBlock := wr.getFromCacheAsUint64(core.MetricNumTxInBlock)
+	rows[1] = []string{fmt.Sprintf("Nr Transactions in block: %v", numTransactionInBlock)}
+
+	numMiniBlocks := wr.getFromCacheAsUint64(core.MetricNumMiniBlocks)
+	rows[2] = []string{fmt.Sprintf("Nr Miniblocks in block: %v", numMiniBlocks)}
+
+	rows[3] = make([]string, 0)
+
+	//Check if node is in metachain
+	//Is in metachain -> display all shards nonce
+	//Is not in metachain -> display metachain nonce
+	rows[4] = []string{fmt.Sprintf("Cross check block height: ")}
```

Comment:
Maybe add a string metric for cross check block height.

Reply:
+1

Reply:
added a new metric for cross check block height.